change_log/next/FE-1244-storybook.yml

```diff
@@ -0,0 +1,3 @@
+New Features: "Adds a storybook"
+New Features: "Adds addons (`Actions`, `Knobs`, `Notes`, `Info`, `ViewPort`)"
+New Features: "Stories for `Alert`, `Animated Menu Button`, `App Wrapper`, `Button`, `Button Toggle` and `Button Toggle Group`."
```

Comment:
you can write this as a single change with multi lines:

```
New Features: |
  Adds Storybook

  Enables these addons (`Actions`, `Knobs`, `Notes`, `Info`, `ViewPort`)

  Added stories for `Alert`, `Animated Menu Button`, `App Wrapper`, `Button`, `Button Toggle` and `Button Toggle Group`.
```

Reply:
sure, fixed, thanks